### ISSUE TITLE
Panorama Sample not working 

### ISSUE THREAD

**Issue by rubenteste** [olexale/arkit_flutter_plugin#139]
just download the project from github and try to run it 
@olexale in the list when you click on the panorama (360 photo sample) it opens a new window, but the screen is all white.

**Comment by olexale** [comment]
hi. what iOS device and version you're testing on?

**Comment by rubenteste** [comment]
iphone 7 -> ios 14.4.2 physical device
but I also can't run on the emulator

**Comment by olexale** [comment]
ARKit is not working on emulators. 
okay, and what about other samples? can you see the globe in the `Image Detection` sample?

**Comment by rubenteste** [comment]
yes all good on image detection, and widget projection works too

**Comment by rubenteste** [comment]
@olexale  can u run it? or is just me that show a white screen?

**Comment by olexale** [comment]
Last Saturday I have finished working on null safety and checked all samples. They all worked for me.

**Comment by olexale** [comment]
I found an issue with the plugin. This should be fixed in 1.0.1, could you please check?

**Comment by rubenteste** [comment]
yes now it works perfectly 
thanks :)
